### DISCUSSION TITLE
use_align_attribute error when upgrading to 3.7.0

### DISCUSSION THREAD

**Discussion by dimton** [netbox-community/netbox#14712] [Help Wanted!] [answered]
I'm running Netbox on Ubuntu VM and upgraded from 3.6.7 to 3.7.0. The upgrade was completed and now when I try to browse the website I get the following error:

```
<class 'KeyError'>

'use_align_attribute'

Python version: 3.8.10
NetBox version: 3.7.0
Plugins: None installed
```

Could you please help?

**Comment by candlerb** [comment]
By "browse the website" do you mean just the front page?

The problem may become clear if you provide a full backtrace. Either set EMAIL and ADMINS in configuration.py, in which case the backtrace will be mailed to you; or temporarily set DEBUG=True, in which case it will be visible in the web UI.

(My guess is you've bypassed the virtualenv somehow)

**Comment by jeremystretch** [comment, marked as answer]
This error occurs because you're running an older version of the Markdown library. NetBox v3.7.0 requires Markdown 3.5.1 (per [`requirements.txt`](https://github.com/netbox-community/netbox/blob/develop/requirements.txt)).

You need to follow the [documented upgrade process](https://docs.netbox.dev/en/stable/installation/upgrading/) to ensure all dependencies are upgraded appropriately.

**Reply by dimton** [reply]
You are right, I updated the requirements.txt and reran the installation; now it works! Thanks
